src/attribution/onLCP.ts

```diff
@@ -54,9 +54,13 @@ const attributeLCP = (metric: LCPMetric): LCPMetricWithAttribution => {
               activationStart
           : 0,
       );
-      const lcpResponseEnd = Math.max(
-        lcpRequestStart,
-        lcpResourceEntry ? lcpResourceEntry.responseEnd - activationStart : 0,
+      const lcpResponseEnd = Math.min(
+        // Cap at LCP time (videos continue downloading after LCP for example)
+        lcpEntry.startTime - activationStart,
+        Math.max(
+          lcpRequestStart,
+          lcpResourceEntry ? lcpResourceEntry.responseEnd - activationStart : 0,
+        ),
       );
       const lcpRenderTime = Math.max(
```

Comment:
I just noticed this uses `lcpEntry.startTime - activationStart`, but `metric.value` is also clamped to 0 so it's never negative:

https://github.com/GoogleChrome/web-vitals/blob/9b932519b16f72328c6d8e9814b811f1bc1a0bb5/src/onLCP.ts#L74

Should `lcpRenderTime` do the same thing (I assume we want the subparts to add up to LCP)? Or just use `metric.value` directly

Reply:
In theory you can never get an LCP entry before an `activationStart` (you need to activate to get a paint, to get LCP), though not sure that's guaranteed for non-TAO cross-origin images... 🤔

I don't think we can use `metric.value` that's a duration and we want a timestamp. Unless we do:
`elementRenderDelay = metric.value - resourceLoadDuration - resourceLoadDelay - timeToFirstByte`
But not sure that's any clearer.

Reply:
After this change, `lcpRenderTime` and `metric.value` are the same value as long as they're ≥ 0. I'm trying to clarify if `lcpRenderTime` should ever be negative or should it always match `metric.value`.

Reply:
Oh yes LCP is a timestamp. Getting confused with the INP change going on at the same time (when `metric.value` is not a timestamp!).

Yes it should always be the same. So agree it can be set to `metric.value`